src/module/vue/shared-sheet.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <div class="flexcol">
-    <header class="sheet-header flexrow nogrow" style="gap: 5px">
+    <SheetHeader class="flexrow nogrow" style="gap: 5px">
```

Comment:
Looks like we can drop the `class` attribute here, since the component does `flex-direction: row` and `flex: 0 0 auto`.

```suggestion
    <SheetHeader style="gap: 5px">
```

Reply:
hmm. one thing occurs to me... maybe it'd make more sense to remove the `flex` property from the `SheetHeader` component? and instead have `nogrow` applied when it's used in a layout. rationale: `nogrow` is most cogent to laying it out amongst a bunch of other components, so it's more useful to have it visible there.

Reply:
> but on the other hand a component shouldn't impose the choice of grid or flex layout on its parent.

hmm. YMMV, but for me it comes down to: *is* it a good idea to presume a flex layout? personally, i strongly prefer to use grid for layouts where stuff has to be placed on both the horizontal and vertical axes (in other words, two-dimensionally), which tends to be the case when working at the level of "laying out the major divisions of a character sheet".

a possible downside w/ grid is that it's not *quite* as automatic re: layout at different sizes. but personally i find it easier to just write another grid layout with a CSS media query -- different size factors have different needs, particularly when a relatively complex two-dimensional layout like a character sheet is involved; i reckon the "small-scale layout for free" aspect of flexbox is only useful if the layout it produces is useful, which is much less likely to happen in two-dimensional character sheet layout (since it's fundamentally a tool for one-dimensional layout).

flexbox, on the other hand, is more optimized for stuff running in a single direction. very useful inside grid divisions because it handles some nuances of 1D layout better than grid does.